app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -51,35 +52,43 @@ class Config {
 
   Future<String> get githubOAuthToken => _getSingleValue('GitHubPRToken');
 
-  Future<String> get nonMasterPullRequestMessage => _getSingleValue('NonMasterPullRequestMessage');
+  Future<String> get nonMasterPullRequestMessage =>
+      _getSingleValue('NonMasterPullRequestMessage');
 
   Future<String> get webhookKey => _getSingleValue('WebhookKey');
 
   Future<String> get missingTestsPullRequestMessage =>
       _getSingleValue('MissingTestsPullRequestMessage');
 
-  Future<String> get goldenBreakingChangeMessage => _getSingleValue('GoldenBreakingChangeMessage');
+  Future<String> get goldenBreakingChangeMessage =>
+      _getSingleValue('GoldenBreakingChangeMessage');
 
-  Future<String> get goldenTriageMessage => _getSingleValue('GoldenTriageMessage');
+  Future<String> get goldenTriageMessage =>
+      _getSingleValue('GoldenTriageMessage');
 
   Future<String> get forwardHost => _getSingleValue('ForwardHost');
 
   Future<int> get forwardPort => _getSingleValue('ForwardPort').then(int.parse);
 
   Future<String> get forwardScheme => _getSingleValue('ForwardScheme');
 
-  Future<int> get maxTaskRetries => _getSingleValue('MaxTaskRetries').then(int.parse);
+  Future<int> get maxTaskRetries =>
+      _getSingleValue('MaxTaskRetries').then(int.parse);
 
   Future<String> get cqLabelName => _getSingleValue('CqLabelName');
 
   /// The URL to connect to the Redis instance for this Cocoon instance.
-  /// 
+  ///
   /// For example, "redis://10.0.0.4:6379" is the default URL on AppEngine
   /// projects.
   Future<String> get redisUrl => _getSingleValue('RedisConnectionSpec');
 
   /// The name of the subcache in the Redis instance that stores responses.
-  Future<String> get redisResponseSubcache => _getSingleValue('RedisResponseSubcache');
+  Future<String> get redisResponseSubcache =>
+      _getSingleValue('RedisResponseSubcache');
+
+  Future<String> get waitingForTreeToGoGreenLabelName =>
+      _getSingleValue('WaitingForTreeToGreenLabelName');
 
```

Comment:
(trivial and optional: i'm guessing the changes above this line to this file are dartfmt changes? they sure make the file harder to read. maybe we should run it with a wider line width?)

Reply:
I think we actually were using a wider width and I just lost it.  I'll try again using wider.